application/features/Audio.py

```
@@ -57,13 +60,16 @@ def __del__(self):
         super().__del__()
 
     def connect(self, *args, **kwargs):
+        logger.debug("Audio: Establishing Audio connection")
         return super().connect(*args, **kwargs)
 
     def launch_audio(self):
```

Ruff

Missing return type annotation for public function `launch_audio`

(ANN201)